### PR TITLE
chore(i18n): Add Korean locale translation

### PR DESCRIPTION
**Pull request by yamoo9**
Add Korean translation file locales/ko.json

Closes issue: N/A

## Reviewer checks

**Required fields, to be filled out by PR reviewer(s)**

- [x] Follows the commit message policy, appropriate for next version
- [x] Code is reviewed for security

### CODE REVIEW

**Comment by straker** [comment]
Fantastic! Let me see if I can find someone who can review this.

File: locales/ko.json [hunk 1]
```diff
@@ -0,0 +1,777 @@
...
+		"landmark-main-is-top-level": {
+			"description": "main 랜드마크가 최상위 레벨인지 확인합니다.",
+			"help": "main 랜드마크는 다른 랜드 마크에 포함되지 않아야 합니다."
+		},
+		"landmark-no-duplicate-banner": {
+			"description": "문서에 최대 하나의 banner 랜드마크가 있는지 확인합니다.",
+			"help": "문서에는 둘 이상의 banner 랜드마크가 없어야 합니다."
+		},
+		"landmark-no-duplicate-contentinfo": {
+			"description": "문서에 최대 하나의 contentinfo 랜드 마크가 있는지 확인합니다.",
```
Comment (jshcrowthe) [inline locales/ko.json]: Your spelling for 랜드 마크 differs from here and most other usages (i.e. 랜드마크). Is that intentional?

File: locales/ko.json [hunk 2]
```diff
@@ -0,0 +1,777 @@
...
+			"description": "p 요소를 스타일링 하여 제목처럼 사용했는지 확인하세요.",
+			"help": "p 요소를 Bold, italic 또는 글자 크기를 조정해 제목처럼 보이게 사용해서는 안됩니다."
+		},
+		"page-has-heading-one": {
+			"description": "페이지 또는 해당 프레임 중 하나 이상의 제목 레벨이 1인 요소가 포함되어 있는지 확인합니다.",
+			"help": "페이지는 제목 레벨이 1인 요소를 포함해야 합니다."
+		},
+		"radiogroup": {
+			"description": "관련 <input type=\"radio\"> 요소에 그룹이 있고 그룹 지정이 일관 되도록 합니다.",
+			"help": "ame 속성 값이 동일한 radio 입력은 그룹의 일부여야 합니다."
```
Comment (jshcrowthe) [inline locales/ko.json]: `name` misspelled here.

File: locales/ko.json [hunk 3]
```diff
@@ -0,0 +1,777 @@
...
+		"page-has-heading-one": {
+			"pass": "페이지에 하나 이상의 제목 레벨 1이 있습니다.",
+			"fail": "페이지에는 제목 레벨 1이 있어야 합니다."
+		},
+		"page-has-main": {
+			"pass": "문서에 하나 이상의 main 랜드마크가 있습니다.",
+			"fail": "문서에 main 랜드마크가 없습니다."
+		},
+		"page-no-duplicate-banner": {
+			"pass": "문서에 둘 이상의 banner 랜드 마크가 없습니다.",
```
Comment (jshcrowthe) [inline locales/ko.json]: 랜드마크?

File: locales/ko.json [hunk 4]
```diff
@@ -0,0 +1,777 @@
...
+			"fail": "문서에 둘 이상의 banner 랜드 마크가 있습니다."
```
Comment (jshcrowthe) [inline locales/ko.json]: 랜드마크?

File: locales/ko.json (hunk 1 above)
Comment (yamoo9) [reply locales/ko.json]: You are right! :-)

'랜드 마크' is wrong.
It's a '랜드마크'.

File: locales/ko.json (hunk 3 above)
Comment (yamoo9) [reply locales/ko.json]: I'm sorry.
"랜드마크" is correct.

File: locales/ko.json (hunk 4 above)
Comment (yamoo9) [reply locales/ko.json]: Yes.
"랜드마크" is correct.

**Comment by straker** [comment]
Thanks @yamoo9 for the translation and @jshcrowthe for the review.

**Pull request merged**